cypress/support/lpa-commands.js

```diff
@@ -11,3 +11,10 @@ Cypress.Commands.add("createLpa", (donorId) =>
       cy.postToApi(`/api/v1/donors/${donorId}/lpas`, lpa).its("body")
     )
 );
+
+Cypress.Commands.add("createDraft", (donorId, lpaId) =>
+  cy.postToApi(
+    `/api/v1/lpas/${lpaId}/documents/draft`,
+    `{"templateId": "IT-AT-LPA", "inserts": ["IT-11"], "correspondentId": ${donorId}}`,
+  ).its("body")
+);
```

Comment:
I doubt we're going to need this again, so might be clearer to just move it into the test rather than have a support function?

Reply:
good point